bigquery/query_builder.py

```diff
@@ -84,14 +101,23 @@ def _render_select(selections):
         original_name = name
         for options_dict in options:
             name = original_name
+            name_qualifiers = list()
+            aggregation_level = options_dict.get('aggregation_level')
```

Comment:
Not thrilled about this make-a-list-and-append structure, but you're sorta forced into it with this whole options dict notion